metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/BigtableMetricModule.java

```diff
@@ -88,7 +89,8 @@ public BigtableMetricModule(
         @JsonProperty("disableBulkMutations") Optional<Boolean> disableBulkMutations,
         @JsonProperty("flushIntervalSeconds") Optional<Integer> flushIntervalSeconds,
         @JsonProperty("batchSize") Optional<Integer> batchSize,
-        @JsonProperty("fake") Optional<Boolean> fake
+        @JsonProperty("fake") Optional<Boolean> fake,
+        @JsonProperty("defaultFetchSize") Optional<Integer> defaultFetchSize
```

Comment:
I'd say, make this "fetchSize", to match the naming of other things such as batchSize. I'm guessing that you chose to name the configuration here like this because it can be overridden on a per-request basis, but I'd say that's a side feature. I'd prioritise keeping the naming in the configuration file uniform.